connectivity/check/result.go

```diff
@@ -167,19 +167,20 @@ func assertMetricsIncrease(metrics ...string) assertMetricsFunc {
 	return func(before, after map[string]*dto.MetricFamily) error {
 		var err error
 		for _, metricName := range metrics {
-			bValue, ok := before[metricName]
+			afterValue, ok := after[metricName]
```

Comment:
it makes i bit difficult to review because variable names changed, and also the order of checking `afterValue` and `beforeValue` flipped. could you keep the same order?

Reply:
Yes getting back to the original order! Thanks :)